### ISSUE TITLE
RubberDuckVBA Unit Tests do not run in Excel files with a space in the file name

### ISSUE THREAD

**Issue by contracta** [rubberduck-vba/Rubberduck#2742]

**Comment by comintern** [comment]
Thanks for taking the time to submit a bug report.  This issue was identified after the release of version 2.0.11 (see #2492), and was fixed with pull request #2494. If you have Visual Studio and are comfortable with [building the project from the source code](https://github.com/rubberduck-vba/Rubberduck/wiki/Contributing), you can pull the current repository and get this fix along with numerous other bug fixes and performance improvements. Otherwise, we're getting close a release of version 2.0.12 (in the next week or so) - so just keep an eye on the releases page.

**Comment by contracta** [comment]
**Computers**
I executed the tests on a desktop and laptop with these configuration details:
Windows 10 - 64 bit 
Office 2007 32 bit 
RubberDuck 2.0.11.2453

**Problem: Rubber Duck Unit Tests do not run in Excel files with a space in the name**
I created a workbook with the name: "RubberDuckVBA UnitTest.xlsb"
I inserted one test module and one simple test:
	Assert.AreEqual "Y", "N"

When I ran the Unit Test I was presented with these error messages:
![xlsx not found](https://cloud.githubusercontent.com/assets/26027799/23332974/13e830d8-fb7b-11e6-9972-d2d123c45857.png)
![htm not found](https://cloud.githubusercontent.com/assets/26027799/23332984/40e6abb4-fb7b-11e6-9b31-b93cd38ef62d.png)

Rubber Duck Unit Test Explorer continued to display the circle of dots indicating that processing was in progress, but it never completed.
I logged off and logged on again. 

Then I opened the RubberDuckVBA UnitTest.xlsb and:
⦁	Went into the VBA editor, 
⦁	Clicked the RubberDuck "Pending" button
⦁	Waited for it to change to "Ready"
⦁	Opened the Test Explorer
⦁	Clicked Run All Tests

No error message boxes were displayed.
Rubber Duck Unit Test Explorer continued to display the circle of dots indicating that processing was in progress, but it never completed.

**Solution**
I logged off and logged on again. 
I renamed "RubberDuckVBA UnitTest.xlsb" to "RubberDuckVBA_UnitTest.xlsb"
**Note:** The change of the space in the file name to an underscore.

Then I opened the RubberDuckVBA_UnitTest.xlsb and:
⦁	Went into the VBA editor, 
⦁	Clicked the RubberDuck "Pending" button
⦁	Waited for it to change to "Ready"
⦁	Opened the Test Explorer
⦁	Clicked Run All Tests.

The tests ran immediately without problem.

Here is a copy of the Excel .xlsb file that exhibited the described problems:
[RubberDuckVBA UnitTest.zip](https://github.com/rubberduck-vba/Rubberduck/files/801339/RubberDuckVBA.UnitTest.zip)

While I have identified a workaround i.e. do not have spaces in Excel file names, it took many hours to identify the problem and it will be quite annoying for Excel users to encounter this error as it appears that RubberDuckVBA does not work.
